skyportal/tests/api/test_shifts.py

```diff
@@ -8,17 +8,18 @@ def test_shift(public_group, super_admin_token, view_only_token, super_admin_use
         'group_id': public_group.id,
         'start_date': '3021-02-27T00:00:00',
         'end_date': '3021-07-20T00:00:00',
+        'description': 'the Night Shift',
+        'shift_admins': [super_admin_user.id],
     }
-
-    status, data = api('POST', 'shift', data=request_data, token=view_only_token)
+    status, data = api('POST', 'shifts', data=request_data, token=view_only_token)
     assert status == 401
     assert data['status'] == 'error'
 
-    status, data = api('POST', 'shift', data=request_data, token=super_admin_token)
+    status, data = api('POST', 'shifts', data=request_data, token=super_admin_token)
     assert status == 200
     assert data['status'] == 'success'
 
-    status, data = api('GET', f'shift/{public_group.id}', token=super_admin_token)
+    status, data = api('GET', f'shifts/{public_group.id}', token=super_admin_token)
     assert status == 200
     assert data['status'] == 'success'
```

Comment:
Maybe check that the shift you posted is in fact recovered. I'd add a uuid name for the shift and make sure you can find that name among one of the returned shifts.

Reply:
That's a great idea, i'll do that now